app/components/EditComponent.js

```diff
@@ -3,23 +3,39 @@ import React, { Component } from 'react';
 import {
   Dimensions,
   Image,
+  Picker,
   StyleSheet,
   View,
 } from 'react-native';
 
 import Button from 'react-native-button';
+import ModalPicker from 'react-native-modal-picker'
+
+const IMAGE_EXPIRATION_LIMIT = 24; // hour
 
 export default class EditComponent extends Component {
 
-  constructor(props) {
-      super(props);
-  }
+  // Initial State
+  state = {
+    expiryTime: 12,
+  };
 
   _onAccept = () => {
     // TODO (rageandqq): something useful
   }
 
   render() {
+    const data = [{
+      key: 0,
+      section: true,
+      label: 'Select Photo Expiration',
+    }];
+    for (let i = 1; i <= IMAGE_EXPIRATION_LIMIT; i++) {
```

Comment:
May as well do all of this computation outside of the render method, above the component